README.md

```diff
@@ -141,6 +141,14 @@ The default tag does not contains the latest API version. Please make sure the l
 
 To fix this error. You should add the latest API version swaggers to the default tag or change default tag to reference latest api version tag.
 
+### MULTIPLE_DEFAULT_TAGS
+
+Level: ERROR
+
+There are more than one default tag.
+
+To fix this error, you should remove other tags and leave only one default tag. You may need to separate into several RP folders or sub folders.

```

Comment:
The expectation is there is only one default tag, which leads to one SDK package.  To release separate SDK packages upon different service resources of the same RP, may consider adopting [Folder Structure for Service Group](https://github.com/Azure/azure-rest-api-specs/blob/main/README.md#folder-structure-for-service-group) , which supports a readme configuration file under each sub folder.